content/scout/policy/_index.md

```diff
@@ -293,19 +293,28 @@ An asterisk (`*`) matches up until the character that follows, or until the end
 of the image reference. Note that the `docker.io` prefix is required in order
 to match Docker Hub images. This is the registry hostname of Docker Hub.
 
-You can also configure the policy to allow only supported tags of Docker
-Official Images. When this option is enabled, images using unsupported tags of
-official images trigger a policy violation. Supported tags for official images
-are listed in the **Supported tags** section of the repository overview on
-Docker Hub.
+You can also configure the policy to:
+
+- Allow only supported tags of Docker Official Images.
+
+  When this option is enabled, images using unsupported tags of official images
+  trigger a policy violation. Supported tags for official images are listed in
+  the **Supported tags** section of the repository overview on Docker Hub.
+
+- Allow only Docker Official Images of supported distro versions
+
+  When this option is enabled, images using unsupported Linux distributions
+  that have reached end of life (such as `ubuntu:18.04`) trigger a policy violation.
+  This check only applies to Docker Official Images.
```

Comment:
This is not true. It will apply to any base images used, so if B depends on A which depends on alpine, we can still track that transitive OS usage.